index.html

```diff
@@ -4677,98 +4722,165 @@ <h2>Internationalization Considerations</h2>
-to be able to explicitly encode the text direction. JSON and its derived formats
-use the UTF-8 encoding of [[Unicode]] for serializing and transmitting text.
-This enables text direction to be directly determined in simple cases,
-but the mechanism is insufficient for general use.
+The next example expresses the human-readable text "revoked" in the Arabic
+language with a base direction of right-to-left:
       </p>
 
+      <pre class="example nohighlight" title="Arabic text with a base direction of right-to-left">
+"statusReason": {
+  "value": "<span class="highlight">إلغاء</span>",
+  "language: "<code>ar</code>-x-dir-<code>rtl</code>"
+}
+      </pre>
+
       <p>
-[[!JSON-LD]] supports the expression of text layout information using the
-<code>rdf:HTML</code> string literal type, as described in [[!JSON-LD]] 1.0,
+Utilization of the design pattern above assumes that the JSON-LD
+<a href="#extensibility">extension</a> associated with the
+<a>property</a> utilizes the Scoped Context feature of JSON-LD to
+alias the <code>@language</code> and <code>@value</code> keywords to
+<code>language</code> and <code>value</code>, respectively. An example of
+a JSON-LD Context snippet doing so is shown below:
+      </p>
+
+      <pre class="example nohighlight" title="Specifying scoped aliasing for language information">
+"statusReason": {
+  <span class="highlight">"@context": {"value": "@value", "language": "@language"}</span>,
+  "@id": "https://www.w3.org/2018/credentials/examples#statusReason"
+}
+      </pre>
+
+    </section>
+
+    <section class="informative">
+       <h3>Complex Language Markup</h3>
+
+       <p>
+When multiple languages, base directions, and annotations are used in a single
+natural language string, more complex mechanisms are typically required.
+[[!JSON-LD]] supports the expression of language information using the
+<code>rdf:HTML</code> string literal type, as described in [[!JSON-LD]],
 <a href="https://www.w3.org/TR/json-ld11/#typed-values">Section 4.2.1: Typed Values</a>.
 This mechanism enables developers to use the internationalization features
-present in HTML to express language information. For example, consider the
-following key-value pair:
+present in HTML to express a variety of language and base direction information.
+For example, consider the following key-value pair that expresses two languages
+in the same natural language text string:
 
-      <pre>
-"nameHtml": "&lt;span dir="rtl" lang="ar">HTML و CSS: تصميم و إنشاء مواقع الويب&lt;/span>"
-      </pre>
+       <pre class="example nohighlight" title="Expressing multiple languages in a single natural language text string">
+"nameHtml": "&lt;span <span class="highlight">lang="en"</span>>Kazumi&lt;/span> (&lt;span <span class="highlight">lang="jp"</span>>和美&lt;/span>)"
+       </pre>
 
-      <p>
+       <p>
+If the JSON-LD Context expressed that any value associated with the
+<code>nameHtml</code> property as being of type <code>rdf:HTML</code>, then
```

Comment:
`as being of type`
-> `is of type`

Reply:
Rewrote the section, which dropped this text entirely.